secant.xcodeproj/project.pbxproj

```diff
@@ -2099,7 +2099,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "swiftlint_version=0.50.3\n\nif which swiftlint >/dev/null; then\n  if ![ $(swiftlint version) = $swiftlint_version ]; then\n    echo \"warning: Compatible SwiftLint version not installed, download version $swiftlint_version from https://github.com/realm/SwiftLint. Currently installed version is $(swiftlint version)\"\n  fi\n   \n  swiftlint \nelse\n  echo \"warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint\"\nfi\n";
+			shellScript = "echo \"Linting the Secant code\"\n./Tools/swiftlint lint --config .swiftlint.yml\necho \"Linting tests\"\n./Tools/swiftlint lint --config .swiftlint_tests.yml\n\n";
```

Comment:
Let's hold off the binary thing until we sort it out by finding an alternative or ignoring the problem altogether because it's safer and more convenient than commiting the binary